app/ui-react/packages/ui/src/Connection/ConnectionCard.tsx

```diff
@@ -242,7 +242,7 @@ export class ConnectionCard extends React.PureComponent<
             <Card.Body>
               <div className={'connection-card__body'}>
                 <div className="connection-card__icon">
-                  <img src={this.props.icon} alt={this.props.name} width={46} />
+                  {this.props.icon && this.props.icon}
```

Comment:
Is this && needed?

Reply:
lemme check if I changed that to be required or not.